specs/FooterMenuItemTemplate-spec.md

```diff
@@ -0,0 +1,123 @@
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+In order to apply a data template to all items in the footer menu of your NavigationView, use the `NavigationView.FooterMenuItemTemplate` property. This property is similar to [NavigationView.MenuItemTemplate](https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.controls.navigationview.menuitemtemplate?view=winui-2.5), but only applies to items placed in NavigationView's footer menu. It takes in a `DataTemplate` and applies that template to any Navigation item placed in the footer menu.
+
+To apply different data templates to individual Navigation items within your footer menu based on certain criteria, use the `NavigationView.FooterMenuItemTemplateSelector`. This property is similar to the [NavigationView.MenuItemTemplateSelector](https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.controls.navigationview.menuitemtemplateselector?view=winui-2.5), but only applies to items placed in NavigationView's footer menu.
+
+The [NavigationView.MenuItemTemplate](https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.controls.navigationview.menuitemtemplate?view=winui-2.5) will only apply to Navigation items that are placed in the main menu of a NavigationView. 
+
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+Example showing how to set `NavigationView.FooterMenuItemTemplate` in markup:
+
+```xml
+<muxc:NavigationView x:Name="NavView" SelectionChanged="NavView_SelectionChanged">
+    <muxc:NavigationView.MenuItemTemplate>
+        <DataTemplate>
+            <muxc:NavigationViewItem Icon="Play" AutomationProperties.Name="MenuItemFromTemplate" Content="{Binding Content}"/>
+        </DataTemplate>
+    </muxc:NavigationView.MenuItemTemplate>
+    <muxc:NavigationView.FooterMenuItemTemplate>
+        <DataTemplate>
+            <muxc:NavigationViewItem Icon="Stop" AutomationProperties.Name="FooterMenuItemFromTemplate" Content="{Binding Content}"/>
+        </DataTemplate>
+    </muxc:NavigationView.FooterMenuItemTemplate>
+    <StackPanel>
+        <TextBlock x:Name="SelectedItem"/>
+    </StackPanel>
+</muxc:NavigationView>
+
+```
```

Comment:
Not sure if this example is a good showcase, I mainly used this for debugging and set the UIA name as a form of a test hook to verify rendering. How about something like a template with child collection being set for the MenuItemsTemplate (indicating deeply nested navigation) and a more simply template (only binding content and image) for the footer items (e.g. this could be a list of accounts).